add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The HTML and PHP code are mixed, which can lead to maintenance challenges and is not a best practice.

Recommendations:
- **Separate Concerns**: Use a templating engine like Twig or Blade to separate PHP logic from HTML.
- **Example**:
  ```php
  // In add_book.php
  $bookRepository = new BookRepository($connection);
  if ($_SERVER['REQUEST_METHOD'] == 'POST') {
      $title = $_POST['title'];
      $author = $_POST['author'];
      $year = $_POST['year'];

      if ($bookRepository->addBook($title, $author, $year)) {
          $message = "Book added successfully!";
      } else {
          $message = "Error: " . $connection->error;
      }
  }
  include 'views/add_book.php';
  ```
  ```html
  <!-- In views/add_book.php -->
  <!DOCTYPE html>
  <html>
  <head>
      <title>Add Book</title>
  </head>
  <body>
      <h1>Add a New Book</h1>
      <?php if (isset($message)) echo "<p>$message</p>"; ?>
      <form method="POST">
          <label>Title:</label>
          <input type="text" name="title"><br>
          <label>Author:</label>
          <input type="text" name="author"><br>
          <label>Year:</label>
          <input type="number" name="year"><br>
          <button type="submit">Add Book</button>
      </form>
  </body>
  </html>
  ```